index.md

```diff
@@ -3,7 +3,7 @@ layout: frontpage
 
 # Header texts
 headerTitle: "The Scala Programming Language"
-headerSubtitle: "Scala is a concise high-level programming language. Scala's compiler helps you avoid bugs when writing complex applications, and its fast runtime lets you build efficient, high-performance systems, with easy access to a huge ecosystem of libraries."
+headerSubtitle: "Scala combines object-orientation with functional programming into one concise, high-level language. Scala's type system helps avoid bugs in complex applications, and its JVM target lets you build high-performance systems with easy access to a huge ecosystem of libraries."
```

Comment:
I think it sounds better to start with "Scala is ..." and state some of the main goals of the language, something like:

"Scala is a concise, statically typed, high-level programming language supporting object-oriented and functional styles."

Reply:
"[...] and it runs on the JVM, yielding high performance and easy access to a huge ecosystem of libraries."